frank/utilities.py

```diff
@@ -347,6 +348,11 @@ def normalize_uv(u, v, wle):
     logging.info('  Normalizing u and v coordinates by provided'
                  ' observing wavelength of {} m'.format(wle))
 
+    if type(wle) is not float or int:
+        if len(wle) != len(u):
+            raise ValueError("len(wle) = {}. It should be"
+                             "equal to len(u) = {}".format(len(wle), len(u)))
+
```

Comment:
```suggestion
    wle = np.array(wle, dtype='f'8)
    if  len(wle) != 1 and len(wle) != len(u):
        raise ValueError("len(wle) = {}. It should be equal to "
                         "len(u) = {} (or 1 if all wavelengths are the same)".format(len(wle), len(u)))
```
This version handles lists properly, and numpy will do the type checking.

Reply:
Perfect, thank you.